packages/link/__stories__/Link.stories.js

```diff
@@ -11,8 +11,29 @@ const Template = (args) => <Link {...args}>{args.children}</Link>;
 export const Default = Template.bind({});
 Default.args = {
   children: 'Go to Code Studio',
-  external: false,
   href: 'https://studio.code.org',
-  openInNewTab: true,
```

Comment:
previously, i added these so they would be displayed in storybook (any missing props were ignored, so the props table wasn't great documentation). now that the DocsPage addon is working, we don't need to do this, and we can exclude the `argTypes` definition above